vmdb/app/controllers/ems_common.rb

```diff
@@ -519,7 +519,7 @@ def build_dc_tree
     ems_node = TreeNodeBuilder.generic_tree_node(
       "ems-#{to_cid(@ems.id)}",
       @ems.name,
-      "ems.png",
+      "#{@ems.type.downcase.sub('ems', 'vendor-')}.png",
```

Comment:
This is not right as downcase is the wrong method...prefer `.image_name` from the model...

```ruby
"vendor-#{@ems.image_name}.png"
```